swan-cmd/src/main/java/de/fraunhofer/iem/swan/model/ModelEvaluator.java

```diff
@@ -97,8 +98,10 @@ public void processResults(SrmList srmList) {
                 }
 
                 try {
-                    if (!options.getOutputDir().isEmpty())
+                    if (!options.getOutputDir().isEmpty()){
+                        predictedSrmList.addMethods(InvokedMethodsFilter.getKnownInvokedMethods());
```

Comment:
Is this null safe?

Reply:
Added null check to SrmList.addMethods().